SPECS-EXTENDED/plexus-component-api/plexus-component-api.spec

```diff
@@ -38,7 +38,7 @@ BuildRequires:  fdupes
 BuildRequires:  java-devel >= 1.6
 BuildRequires:  javapackages-local-bootstrap
 BuildRequires:  plexus-classworlds
-Requires:       mvn(org.codehaus.plexus:plexus-classworlds)
+Requires:       plexus-classworlds
```

Comment:
Did we check an install of the package (with the generated rpm)?
Also, if possible we should verify if the package behaves as expected ?